client/components/theme/index.jsx

```diff
@@ -152,9 +136,10 @@ export class Theme extends Component {
 	render() {
 		const { active, price, theme, translate, upsellUrl } = this.props;
 		const { name, description, screenshot } = theme;
+		const isActionable = !! this.props.screenshotClickUrl || this.props.onScreenshotClick;
```

Comment:
nit: the `!!` cast to boolean is not really necessary there. `classNames` is happy with any truthy/falsy value.